tests/integration/multicluster/loadbalancing.go

```diff
@@ -0,0 +1,44 @@
+package multicluster
+
+import (
+	"fmt"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/features"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+func LoadbalancingTest(t *testing.T, ns namespace.Instance, features ...features.Feature) {
+	framework.NewTest(t).
+		Label(label.Multicluster).
+		Features(features...).
+		Run(func(ctx framework.TestContext) {
+			ctx.NewSubTest("reachability").
+				Run(func(ctx framework.TestContext) {
+					builder := echoboot.NewBuilder(ctx)
```

Comment:
do we need to spin up N echo instances? or can we reuse with other tests

Reply:
Made telemetry/reachability share. Couldn't get that to work for this one and cluster-local without making things slower. 

Now we deploy them all in parallel during suite setup so that should help.